### ISSUE TITLE
Thermostat temperature as an entity

### ISSUE THREAD

**Issue by margit73** [natekspencer/hacs-vivint#157]
### The problem

Hi,
Thank you for this integration.
Is it possible to have thermostat temperature as an entity in Home Assistant, to control it and adjust based on triggers ?

### What version of Home Assistant Core are you running?

2024.11.1

**Comment by natekspencer** [comment]
This should already be handled with the climate entity. I don't have a thermostat with Vivint to test though. @kitprice, since you're fixing a couple of Climate issues at the moment, can you validate this is working as intended for you?

**Comment by kitprice** [comment]
@natekspencer Yes, I was able to use climate.set_temperature to change the temperature settings.  I tested both in dual mode (heat_cool) and single mode (heat or cool).  Note that I do have the older style thermostat (CT100) if that makes a difference.

**Comment by margit73** [comment]
Thank you kitprice for the hint. I was able to change the temperature
through action: climate.set_temperature :)

Is it possible to read just the current temperature from a thermostat
entity and present it as stand alone entity on a dashboard?

**Comment by kitprice** [comment]
@margit73 You will need to create a helper.  Just use your thermostat entity name (from existing climate entity) for the state template and the created sensor will be the entity id (you define).

Entity ID:
sensor.thermostat_current_temperature

State template:
state_attr('climate.thermostat', 'current_temperature')

**Comment by margit73** [comment]
Thank you! This is helpful :)
